android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -107,6 +107,7 @@ public void run() {
       emptyGroupIdQueue();
       mSecondScanComplete = true;
       mSwipeRefreshWidget.setRefreshing(false);
+      mScanningAnimationTextView.setAlpha(0f);
```

Comment:
I think it only needs to be called when the animation has stopped and there are no results. If we call it in the first timeout there would be results so its not needed. Also in `startScanningDisplay ` I don't think its need because if it hits the else the timeouts will be firing soon. I can definitely add it to the `startScanningDisplay ` if you think its makes it clearer.